www-client/seamonkey/seamonkey-2.53.20.ebuild

```diff
@@ -0,0 +1,553 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+WANT_AUTOCONF="2.1"
+
+PYTHON_COMPAT=( python3_{10..11} )
+PYTHON_REQ_USE='ncurses,sqlite,ssl,threads(+)'
+
+LLVM_COMPAT=( {17..19} )
+
+# This list can be updated with scripts/get_langs.sh from the mozilla overlay
+# note - could not roll langpacks for: ca fi
+#MOZ_LANGS=(ca cs de en-GB es-AR es-ES fi fr gl hu it ja lt nb-NO nl pl pt-PT
+#	    ru sk sv-SE tr uk zh-CN zh-TW)
+MOZ_LANGS=(cs de en-GB es-AR es-ES fr hu it ja lt nl pl pt-PT
+	    ru sk sv-SE zh-CN zh-TW)
+
+MOZ_PV="${PV}"
+MOZ_PV="${MOZ_PV/_beta/b}"
+MOZ_P="${P}"
+MY_MOZ_P="${PN}-${MOZ_PV}"
+PATCH_PV="2.53.20"
+PATCH="${PN}-${PATCH_PV}-patches"
+PATCH_S="${WORKDIR}/gentoo-${PN}-patches-${PATCH_PV}"
+
+DESCRIPTION="Seamonkey Web Browser"
+HOMEPAGE="https://www.seamonkey-project.org/"
+
+SRC_URI="https://archive.seamonkey-project.org/releases/${MOZ_PV}/source/${MY_MOZ_P}.source.tar.xz
+	https://archive.seamonkey-project.org/releases/${MOZ_PV}/source/${MY_MOZ_P}.source-l10n.tar.xz
+	https://github.com/BioMike/gentoo-${PN}-patches/archive/refs/tags/${PATCH_PV}.tar.gz -> ${PATCH}.tar.gz"
+
+S="${WORKDIR}/${MY_MOZ_P}"
+
+MOZ_GENERATE_LANGPACKS=1
+MOZ_L10N_SOURCEDIR="${S}/${P}-l10n"
+inherit autotools check-reqs desktop edos2unix flag-o-matic llvm-r1 mozcoreconf-v6 mozlinguas-v2 pax-utils \
+	rust toolchain-funcs xdg-utils
+
+LICENSE="MPL-2.0 GPL-2 LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+SYSTEM_IUSE=( +system-{av1,harfbuzz,icu,jpeg,libevent,libvpx,png,sqlite} )
+IUSE="+chatzilla cpu_flags_arm_neon dbus +gmp-autoupdate +ipc jack
+lto pulseaudio selinux startup-notification test webrtc wifi"
+IUSE+=" ${SYSTEM_IUSE[@]}"
+
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	app-arch/unzip
+	app-arch/zip
+	>=dev-lang/nasm-2.13
+	dev-lang/perl
+	dev-util/cbindgen
+	>=sys-devel/binutils-2.16.1
+	$(llvm_gen_dep '
+		llvm-core/clang:${LLVM_SLOT}=
+		llvm-core/llvm:${LLVM_SLOT}=
+	 ')
+	virtual/pkgconfig
+	amd64? ( >=dev-lang/yasm-1.1 )
+	lto? ( sys-devel/binutils[gold] )
```

Comment:
New binutils doesn't have the gold flag anymore, as gold support is finally deprecated and removed. You'll have to adjust for that.

Although I see lto is use-masked for seamonkey anyway. But could you get rid of this vestige?